src/pages/Facility/overview.tsx

```diff
@@ -57,14 +59,17 @@ export function FacilityOverview({ facilityId }: FacilityOverviewProps) {
   ];
 
   return (
-    <div className="">
-      <div className="container p-6 mx-auto space-y-8">
+    <Page title="">
```

Comment:
```suggestion
    <Page>
```

is this prop `title` needed? why not make it optional